packages/core/src/components/Card/Card.tsx

```diff
@@ -51,10 +51,11 @@ export const HvCard = ({
       aria-selected={selectable ? selected : undefined}
       className={clsx(
         styles.root,
-        bgcolor &&
-          css({
-            backgroundColor: theme.colors[bgcolor],
-          }),
+        // bgcolor &&
+        css({
+          backgroundColor:
+            theme.card.backgroundColor || (bgcolor && theme.colors[bgcolor]),
+        }),
```

Comment:
I think by doing this the background set in the theme will always be used instead of the one passed to the component through the prop `bgcolor`

Reply:
Ah, you're right. Will fix it!

Reply:
Yes, it also makes sense for me